app/views/items/browse_facet.html.erb

```diff
@@ -42,21 +42,30 @@
     <div role="tabpanel"
       class="tab-pane <%= active?(facet_name, @browse_facet) %>" id="<%= id %>">
       <% if @res.key?(facet_name) %>
-        <ul class="list-unstyled">
-          <% @res[facet_name].each do |list| %>
-            <li class="row">
-              <%= link_to search_path("f" => ["#{facet_name}|#{list[0]}"]) do %>
-                <% text = list[0].blank? ? "No label" : list[0] %>
-                <div class="col-lg-5 col-md-6 col-sm-8 col-xs-10 browse-item">
```

Comment:
Could you also remove the `.browse-item` class from the SCSS? Otherwise looks great from reviewing code. I'll pull it down and test it in my basic Orchid app